### ISSUE TITLE
Fails when not using a display manager 

### ISSUE THREAD

**Issue by Brottweiler** [gilbertw1/rofellite#5]
or at least it is not showing up with rofi anymore. Running in a terminal just gives a terminal interface.

**Comment by matthewcm** [comment]
Running in terminal uses interactive fuzzy finding. However if you launch roficlip normally it will execute on the rofi gui.

I am using BSPWM as my window manager and have this:
```ini
# Clipboard
super + c
   roficlip
```

Summary: To get the visual interface, it must be run outside of a terminal

**Comment by Brottweiler** [comment]
Right, my post might have been easy to missunderstand. I use IceWM, and I got this in my `keys` file.

```bash
key "Super+l"               slock
key "Super+v"               roficlip
key "Super+period"          rofimoji
```
the shortcut for `slock` and `rofimoji` works, but not the one for `roficlip`. This suddenly stopped working for me, but it sounds like there might be some other issue than roficlip itself.

**Comment by Brottweiler** [comment]
@matthewcm I figured out my issue. roficlip only works if I use a display manager. This is why it stopped working for me, because I stopped using a display manager.

When I use roficlip hotkey without a display manager, then I log out from my WM, roficlip has launched in CLI mode in the TTY.

**Comment by sshaikh** [comment]
This was useful and I see it too. Any workarounds?

**Comment by Brottweiler** [comment]
> This was useful and I see it too. Any workarounds?

I think what I did as a workaround was to force it to use rofi. Because when you don't use a display manager, roficlip thinks you are always in a terminal even when you are not, so it tries to run fzf: https://github.com/gilbertw1/roficlip/blob/master/roficlip#L25

I don't use roficlip anymore, but what I recall doing was just to remove line 25-29 and keep
```bash
SELECTION="$(echo "$C_HIST" | rofi -dmenu -i -lines 20 -width 80 -p 'selection: ')"
```
This obviously removes the ability to use it in a terminal, but I never used it in a terminal anyway so it didn't matter for me.

**Comment by nedzhang** [comment]
> This was useful and I see it too. Any workarounds?

Here is my workaround (i3 without display manager) in .config/i3/config:

```
# shortcut to selection widget (primary). Use /dev/null to false the [[ -t 0 ]]  check
bindsym $mod+c exec --no-startup-id "roficlip < /dev/null"
```
